Assets/_BForBoss/_Weapons/Scripts/Weapons/MeleeWeaponBehaviour.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace Perigon.Weapons
+{
+    public class MeleeWeaponBehaviour : MonoBehaviour
+    {
+        [SerializeField] private MeleeScriptableObject _meleeScriptable;
+        [SerializeField] private Transform _playerTransform;
```

Comment:
Can you make a check on Awake, to see if this is null? 
And then Debug.LogWarning()